lib/rules/indent.js

```diff
@@ -61,6 +66,24 @@ module.exports = function(context) {
         }
     }
 
+    if (typeof options.VariableDeclarator === "number") {
+        options.VariableDeclarator = {
+            var: options.VariableDeclarator,
+            let: options.VariableDeclarator,
+            const: options.VariableDeclarator
+        };
+    } else {
```

Comment:
I get that. Since at the top during options declaration you have already assigned defaults to it. Now when you use `assign` it will only override those options which the user provide. Your defaults will remain intact.